exchanges/okx/okx.go

```diff
@@ -1862,27 +1677,83 @@ func (ok *Okx) GetConvertHistory(ctx context.Context, before, after time.Time, l
 		params.Set("tag", tag)
 	}
 	var resp []ConvertHistory
-	return resp, ok.SendHTTPRequest(ctx, exchange.RestSpot, getConvertHistoryEPL, http.MethodGet, common.EncodeURLValues(assetConvertHistory, params), nil, &resp, true)
+	return resp, ok.SendHTTPRequest(ctx, exchange.RestSpot, getConvertHistoryEPL, http.MethodGet, common.EncodeURLValues("asset/convert/history", params), nil, &resp, request.AuthenticatedRequest)
 }
 
 /********************************** Account endpoints ***************************************************/
 
+// AssetTypeString returns a string representation of asset type
+func AssetTypeString(assetType asset.Item) (string, error) {
+	switch assetType {
+	case asset.Spot:
+		return "SPOT", nil
+	case asset.Margin:
+		return "MARGIN", nil
+	case asset.Futures:
+		return "FUTURES", nil
+	case asset.Options:
+		return "OPTION", nil
+	case asset.PerpetualSwap:
+		return "SWAP", nil
+	case asset.Spread:
+		return "SPREAD", nil
+	case asset.LinearContract:
```

Comment:
Looks like asset.LinearContract can be removed as all the other instruments are covered above